searchlib/src/vespa/searchlib/fef/indexproperties.h

```diff
@@ -178,6 +178,18 @@ namespace mutate {
 
 // Add temporary flags used for safe rollout of new features here
 namespace temporary {
+/**
+ * A number in the range [0,1] for the effective idf range for WeakAndOperator.
+ * 1.0 will give the complete range as used by default by bm25.
+ * idf = (1.0 - range) + (range * idf)
```

Comment:
The formula is missing a detail: `scaled_idf = (1.0 - range) * max_idf + (range * idf)`

Reply:
Fixed